Kickstarter-iOS/DataSources/SettingsAccountDataSource.swift

```diff
@@ -46,13 +56,17 @@ final class SettingsAccountDataSource: ValueCellDataSource {
       return value.cellType as? SettingsAccountCellType
     } else if let currencyValue = self[indexPath] as? SettingsCurrencyCellValue {
       return currencyValue.cellType as? SettingsAccountCellType
+    } else if self[indexPath] is Bool {
```

Comment:
Could we follow the same pattern `else if let...` here? I think checking if `self[indexPath] is Bool` get's a little harder to read.

Reply:
Swiftlint was complaining about unused optional binding but I can silence